pkg/cli/serve/config.go

```diff
@@ -21,12 +21,11 @@ func ReadFlagsFromFile(cmd *cobra.Command, names ...string) error {
 				os.Exit(1)
 			}
 
-			if len(values) > 1 {
-				fmt.Fprintf(os.Stderr, "Must only specify one value when reading a string slice from file")
-				os.Exit(1)
-			}
-
 			if strings.HasPrefix(values[0], "@") {
+				if len(values) > 1 {
```

Comment:
@bmeng I moved this because it was happening on flag values like:
```
services "service_logs,clusters"
```
instead of 
```
services "@service_logs,clusters"